extension.js

```diff
@@ -128,6 +129,11 @@ class ReferenceProvider {
         if (!newUriPath.match(/\.js$/i)) newUriPath += '.js';
 
         const newUri = vscode.Uri.file(newUriPath);
+
+        if (!fs.existsSync(newUri.fsPath)) {
```

Comment:
I tried that in my previous PR and could not suppress the error with a try catch hence this solution. I also tried looking into the vscode api documentation as you said its handled as expected. If you really think this (drastically) slows down the plugin I would just leave this bug for now as a known issue.